src/python/CRABInterface/RESTTask.py

```diff
@@ -115,6 +115,16 @@ def webdir(self, **kwargs):
         yield row.user_webdir
 
 
+    def getpublishurl(self, **kwargs):
+        if 'workflow' not in kwargs or not kwargs['workflow']:
+            raise InvalidParameter("Task name not found in the input parameters")
+        workflow = kwargs['workflow']
```

Comment:
I think it's cleaner to just use the `kwargs['workflow']` instead of making a new variable